spring-cloud-modules/spring-cloud-openfeign/src/main/java/com/baeldung/cloud/openfeign/customizederrorhandling/config/FeignConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.cloud.openfeign.customizederrorhandling.config;
+
+import feign.Logger;
+import feign.codec.ErrorDecoder;
+import org.springframework.context.annotation.Bean;
+
+public class FeignConfig {
+
+    @Bean
+    Logger.Level feignLoggerLevel() {
+		return Logger.Level.FULL;
+	}
```

Comment:
The spacing's gone all funny here

Reply:
Ya funny :). I guess side effect of reusing the code already present in the project.